examples/chain.js

```diff
@@ -1,9 +1,9 @@
-var Nemo = require("../");
-var config = {
-  "driver": {
-    "browser": "phantomjs"
-  },
-  'data': {
+const Nemo = require('../');
+const config = {
+  'driver': {
+    'browser': 'phantomjs'
+  }
+  ,'data': {
```

Comment:
Fixes done in d554742f83922e56b3d8eb06343fe31267fae5b1 

Some useful eslint links:
http://eslint.org/docs/rules/comma-dangle
http://eslint.org/docs/rules/comma-spacing
http://eslint.org/docs/rules/comma-style